appveyor.yml

```diff
@@ -12,7 +12,7 @@ init:
   - ps: Start-FileDownload 'http://download.microsoft.com/download/7/9/6/796EF2E4-801B-4FC4-AB28-B59FBF6D907B/VCForPython27.msi' C:\VCForPython27.msi; echo "Done"
   - cmd: msiexec /i C:\VCForPython27.msi /quiet /qn
   - set PATH=C:\Miniconda-x64;C:\Miniconda-x64\\Scripts;%PATH%  # Miniconda is already installed on appveyor: https://github.com/appveyor/ci/issues/359
-  - conda install --yes numpy cython pytables scipy matplotlib nose numba mock pyyaml
+  - conda install --yes numpy cython pytables scipy matplotlib nose numba mock pyyaml pyreadline
```

Comment:
Otherwise [appveyor complains](https://ci.appveyor.com/project/DavidLP/testbeam-analysis/build/1.0.1258#L285) during `dill` installation. But I think this is not really needed, since we do not pickle something in ipython. I delete.